wrappers/Python/setup.py

```diff
@@ -105,18 +107,24 @@ def recursive_collect_includes():
         i = sys.argv.index(cmake_args[0])
         sys.argv.pop(i)
         cmake_compiler, cmake_bitness = cmake_args[0].split('cmake=')[1].split(',')
+    elif os.environ.get('COOLPROP_CMAKE'):
+        cmake_compiler, cmake_bitness = os.environ.get('COOLPROP_CMAKE').split(',')
```

Comment:
Allow getting the cmake compiler and bitness from an env variable, so I can pass it to cibuildwheel

Reply:
Because you need it in the build matrix?